addon/components/mock-login.hbs

```diff
@@ -3,6 +3,7 @@
     errorMessage=this.errorMessage
     loading=this.loginTask.isRunning
     isLoading=this.loginTask.isRunning
+    login=(perform this.loginTask)
```

Comment:
Any thoughts about this? The yielded object is usually aliased as `login` and `login.login` does look a bit weird :smile:.